e2e/e2e_test.go

```diff
@@ -99,42 +106,11 @@ func TestMain(m *testing.M) {
 	}
 	os.Exit(code)
 }
-func TestCreatesInsecureCluster(t *testing.T) {
-	if testing.Short() {
-		t.Skip("skipping test in short mode.")
-	}
-
-	testLog := zapr.NewLogger(zaptest.NewLogger(t))
-
-	actor.Log = testLog
-
-	sb := testenv.NewDiffingSandbox(t, env)
-	sb.StartManager(t, controller.InitClusterReconcilerWithLogger(testLog))
-
-	b := testutil.NewBuilder("crdb").WithNodeCount(3).WithEmptyDirDataStore()
-
-	create := Step{
-		name: "creates 3-node insecure cluster",
-		test: func(t *testing.T) {
-			require.NoError(t, sb.Create(b.Cr()))
 
-			requireClusterToBeReadyEventually(t, sb, b)
-
-			state, err := sb.Diff()
-			require.NoError(t, err)
-
-			expected := testutil.ReadOrUpdateGoldenFile(t, state, *updateOpt)
-
-			testutil.AssertDiff(t, expected, state)
-		},
+func TestCreatesSecureCluster(t *testing.T) {
```

Comment:
this is pretty self explanatory but please add a comment for consistency sake

Reply:
You don't usually add comments on test funcs in go ;). It is outside of the go standard.  I will add some comments inside the func.